### DISCUSSION TITLE
Better state Management for Flutter

### DISCUSSION THREAD

**Discussion by AshNiz24** [community/community#38254] [Other Feature Feedback, Questions, & Ideas] [answered]
Which is the best state management for flutter?

**Comment by chandran-jr** [comment, marked as answer]
Personally I am in favour of getx as it has state management along with easier screen navigation, custom snackbar, alerts and bottom sheets.
But if you are a beginner it is better to start with provider

**Comment by fal3n-4ngel** [comment]
Getx is the most popular one since almost everyone uses it and it's pretty easy n cool.
